config-as-code/environments/ng-health-training.yaml

```
@@ -0,0 +1,786 @@
+digit-ui:
+  custom-js-injection: |
+    sub_filter.conf: "
+      sub_filter  '<head>' '<head>
+      <script src=https://ng-health-uat-static-assets.s3.af-south-1.amazonaws.com/globalConfig.js type=text/javascript></script>
+      ';"
+#>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+egov-filestore:
+  volume: /opt/eGov/filestore
+  is-bucket-fixed: "true"
+  minio.url : "https://s3.amazonaws.com"
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)